net-mail/getmail/getmail-6.03.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
Could we also add Python 3.9 support here? Looking at https://www.python.org/downloads/release/python-390/ it does not immediately seem there are breaking changes between 3.8 and 3.9 (Unicode update might be interesting though), nor does getmail6 home page specify limits on Python3 compatibility.

Since it's a testing package anyway, I'd say let people try it on higher versions of Python.

Reply:
The maintainer should test it, but yeah, it's likely to work.